clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/deploy/description/AbstractCloudFoundryDescription.java

```diff
@@ -18,11 +18,20 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.netflix.spinnaker.clouddriver.cloudfoundry.client.CloudFoundryClient;
+import com.netflix.spinnaker.clouddriver.cloudfoundry.security.CloudFoundryCredentials;
+import com.netflix.spinnaker.clouddriver.security.resources.AccountNameable;
 import lombok.Data;
 
 @Data
-public abstract class AbstractCloudFoundryDescription {
+public abstract class AbstractCloudFoundryDescription implements AccountNameable {
   @JsonIgnore private CloudFoundryClient client;
 
   private String region;
+
+  @JsonIgnore private CloudFoundryCredentials credentials;
+
+  @Override
+  public String getAccount() {
+    return credentials.getName();
```

Comment:
Safety check: check if `credentials` is null before using it, to prevent `NullPointerException` if `getAccount` is ever called before setting the `credentials` object.

Reply:
@german-muzquiz Added some logic here for the safety check!